source/administration/devops-command-center.rst

```diff
@@ -176,3 +184,17 @@ Channel Export
 --------------
 
 Please see the `Channel Export plugin documentation <https://mattermost.gitbook.io/channel-export-plugin>`_ for more information.
+
+Glossary
```

Comment:
Consider listing glossary entries in alphabetical order for easy scan and retrieval.